From af739abf7a4bbe67df058eac0e49374556341b23 Mon Sep 17 00:00:00 2001
From: glibc <glibc>
Date: Thu, 9 Sep 2021 01:44:47 +0000
Subject: [PATCH 2/6] patch misc/regexp.c

---
 misc/regexp.c | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/misc/regexp.c b/misc/regexp.c
index 3b36682..025a2f1 100644
--- a/misc/regexp.c
+++ b/misc/regexp.c
@@ -26,17 +26,18 @@
 
 #include <regex.h>
 #include <shlib-compat.h>
+#include <stdio.h>
 
 #if SHLIB_COMPAT (libc, GLIBC_2_0, GLIBC_2_23)
 
 /* Define the variables used for the interface.  */
-char *loc1;
-char *loc2;
+char *loc1 = NULL;
+char *loc2 = NULL;
 compat_symbol (libc, loc1, loc1, GLIBC_2_0);
 compat_symbol (libc, loc2, loc2, GLIBC_2_0);
 
 /* Although we do not support the use we define this variable as well.  */
-char *locs;
+char *locs = NULL;
 compat_symbol (libc, locs, locs, GLIBC_2_0);
 
 
-- 
2.17.1

