From f66e5ab0eac205ddf9f50be37adc1588ad35aee4 Mon Sep 17 00:00:00 2001
From: glibc <glibc>
Date: Thu, 9 Sep 2021 02:00:26 +0000
Subject: [PATCH 5/6] patch nis/nis_call.c

---
 nis/nis_call.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/nis/nis_call.c b/nis/nis_call.c
index 3fa37e4..2537f2a 100644
--- a/nis/nis_call.c
+++ b/nis/nis_call.c
@@ -679,7 +679,7 @@ nis_server_cache_add (const_nis_name name, int search_parent,
 
   /* Choose which entry should be evicted from the cache.  */
   loc = &nis_server_cache[0];
-  if (*loc != NULL)
+  if (*loc != NULL) {
     for (i = 1; i < 16; ++i)
       if (nis_server_cache[i] == NULL)
 	{
@@ -690,6 +690,7 @@ nis_server_cache_add (const_nis_name name, int search_parent,
 	       || ((*loc)->uses == nis_server_cache[i]->uses
 		   && (*loc)->expires > nis_server_cache[i]->expires))
 	loc = &nis_server_cache[i];
+  }
   old = *loc;
   *loc = new;
 
-- 
2.17.1

